src/mutation/RelayMutationQueue.js

```diff
@@ -350,14 +355,15 @@ class PendingTransaction {
 
   getCollisionKey(): ?string {
     if (this._collisionKey === undefined) {
-      this._collisionKey = this.mutation.getCollisionKey() || null;
+      this._collisionKey =
+        this.mutation.getCollisionKey(this.getProps()) || null;
     }
     return this._collisionKey;
   }
 
   getConfigs(): Array<RelayMutationConfig> {
     if (!this._configs) {
-      this._configs = this.mutation.getConfigs();
+      this._configs = this.mutation.getConfigs(this.getProps());
```

Comment:
Why the change to pass props? It doesn't seem like `getConfigs` method signature has changed. Again, let's split the `RelayMutation*` changes into a (single) separate PR.